### DISCUSSION TITLE
Invoke-VCFHealthReport not working

### DISCUSSION THREAD

**Discussion by johnazevedoca** [vmware/powershell-module-for-vmware-cloud-foundation-reporting#212] [Community Help] [answered]
### Code of Conduct

- [X] I have read and agree to the [Code of Conduct](https://github.com/vmware/powershell-module-for-cloud-foundation-reporting/blob/main/CODE_OF_CONDUCT.md).
- [ ] Vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue initial description to help the maintainers prioritize.
- [ ] Do not leave "+1" or other comments that do not add relevant information or questions.
- [ ] If you are interested in working on this issue or have submitted a pull request, please leave a comment.

### VMware Cloud Foundation

5.0

### PowerShell Version

Windows Powershell 7.2.13

### PowerCLI Version

12.3.0 build 17860403

### Module Version

2.6.0.1001

### PowerValidatedSolutions Version

2.8.0.1048

### PowerVCF Version

2.4.1.1000

### Guest Operating System

Windows Server 2019

### Environment Details

We have a management domain and a Workload domain, they are both in the same version 5.0 and are a stretched cluster with 4 nodes on each AZ.

### Description

[powershellerror.docx](https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-reporting/files/14464681/powershellerror.docx)

 
 
 

### Error or Debug Output

Not sure what to add in here.

### Expected Behavior

I'm expecting to see the report generated

### Actual Behavior

I get the following error.

![image](https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-reporting/assets/160673649/ca087642-82de-412b-8bf6-6c5ba9804328)

### Steps to Reproduce

![image](https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-reporting/assets/160673649/82a92112-ae55-45f2-97d6-1ce20a8830d5)

**Comment by tenthirtyam** [comment, marked as answer]
Based on the error, you're getting an eight getting an HTTP 401 unauthorized error. Please try the Request-VCFToken cmdlet from PowerVCF to see if you're @ae2.ca user is authorized. You can also test with the admin@local API user as seen in the cmdlet examples.

if the token cannot be obtained, the report will fail.

**Reply by tenthirtyam** [reply]
Migrated from the issues to discussions.

**Comment by tenthirtyam** [comment]
Please note that your environment also does not meet the minimum requirements for `VMware.PowerCLI`, which should be at 13.1.0 at a minimum per the module documentation.
